src/puppetlabs/trapperkeeper/internal.clj

```diff
@@ -219,11 +219,15 @@
   "Initializes a 'worker' which will listen for lifecycle-related tasks and perform
   them on a background thread, to ensure that we aren't executing multiple lifecycle
   tasks simultaneously."
-  [lifecycle-chan :- (schema/protocol async-prot/Channel)
+  [lifecycle-channel :- (schema/protocol async-prot/Channel)
+   shutdown-channel :- (schema/protocol async-prot/Channel)
    shutdown-reason-promise :- IDeref]
+  (log/debug "Initializing lifecycle worker loop.")
   (async/go (try
               (loop []
```

Comment:
In thinking about this further, though, I think maybe I need to move that try inside of the loop for a different reason (because if the loop exits then there will be no one left to try to respond to the shutdown message; this was a change in behavior introduced in my previous PR), so I'll see about fixing that in another commit when I update the comment that you mentioned.